lms/djangoapps/shoppingcart/models.py

```diff
@@ -1884,7 +1884,7 @@ class Meta(object):
     course_id = CourseKeyField(unique=True, max_length=128, db_index=True)
     annotation = models.TextField(null=True)
 
-    def __unicode__(self):
+    def __str__(self):
```

Comment:
This will break things that try to cast as `unicode(object)`  You should use the python_2_unicode_compatible decorator on the class: https://docs.djangoproject.com/en/2.2/ref/utils/#django.utils.encoding.python_2_unicode_compatible

Reply:
done

Reply:
Don't see the change on the PR, did you push up the change?

Reply:
done.